src/content/blog/whats-new-february-2024.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: "What's new in Astro - February 2024"
+description: "We started of 2024 strong"
+publishDate: "February 29, 2024"
+authors:
+  - elian
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-february-2024/og.webp"
+lang: "en"
+---
+import ShowcaseGrid from './_whats-new-components/ShowcaseGrid.astro';
+import ThemeGrid from './_whats-new-components/ThemeGrid.astro';
+
+little sentence here @todo:
+
+## Updates from Mission Control
+
+- We have a [new channel on Discord](/chat) `#events`, where you can share events happening, share when you're live, and more.
+
```

Comment:
daily dev thing here too?

Reply:
Yezh! We definitely should!

I was also thinking about a way for ecosystem-ci to be included here?